app/src/main/java/com/allan/boardbuddies/fragments/NotesFragment.java

```diff
@@ -68,4 +78,49 @@ public void onClick(View view) {
             }
         });
     }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        loadNotes();
+        adapter.notifyDataSetChanged();
```

Comment:
This is expensive, but since you know exactly which note you opened, is there a way you can only notify the position that was edited?